### ISSUE TITLE
Adicionar data e hora da realização dos testes

### ISSUE THREAD

**Issue by erickseolin** [erickseolin/jn_tester#29]
Acho legal adicionar a hora em que foi realizado o teste, Assim pode-se ver em que momento os alunos estão utilizando o sistema.

**Comment by rdenadai** [comment]
Concordo... é uma boa métrica para ter uma ideia de quando os testes foram executados.

**Comment by rdenadai** [comment]
Lembrando que na hora de fazer esta implementação... mudar também o arquivo de testes https://github.com/erickseolin/jn_tester/blob/master/tools/results-generator.py para que fique sincronizado com a implementação principal.

**Comment by rdenadai** [comment]
Estou tratando este issue na branch https://github.com/erickseolin/jn_tester/tree/fix_issue_29

**Comment by rdenadai** [comment]
Merge realizado na branch develop... disponibilizado para testes.
